lib/UserName/tests/interactor.js

```diff
@@ -0,0 +1,6 @@
+import { interactor } from '@bigtest/interactor';
+import { Label as LabelInteractor } from '@folio/stripes-testing';
+
+export default interactor(class UserNameInteractor {
+  Label = new LabelInteractor();
```

Comment:
`Label` or `label`? I think the latter since this is effectively an instance property, and that seems to be the convention in other interactors in this module.

Reply:
Agree, going with `label`.